src/for_2D_build/common/data_type.h

```diff
@@ -61,8 +61,11 @@ const Matd reduced_unit_matrix{
 
 /** initial local normal, only works for thin structure dynamics. */
 const Vecd local_pseudo_n_0 = Vecd(0.0, 1.0);
-
 const Vecd ZeroVecd = Vec2d::Zero();
+
```

Comment:
These changes better be updated by merge the main master to your fork other than change them manually. Otherwise it will give conflict for future merge.